README.md

```diff
@@ -196,3 +197,20 @@ module.exports = (obj) => validator.validate(obj, companySchema);
 const schema = require('./user.schema')
 const usersService = db.createService('users', schema);
 ```
+
+In order to add your own methods to the services, you can use the functions `setServiceMethod` and `setQueryServiceMethod` of the object `db`. This function takes the function name as the first parameter and the custom function for the service as the second parameter. The custom function takes the service itself as the first parameter, and the remaining parameters are the parameters that are passed when this custom function is called.
+
+```javascript
+const connectionString = `mongodb://localhost:27017/home-db`;
+const db = require('./').connect(connectionString);
+
+db.setQueryServiceMethod('findById', (service, id) => {

```

Comment:
Well, I don't really like this way of extending. I think the right way is to inherit from the base service and add methods as needed. We can statically set these methods within the factory we expose. What do you think? It might be too much OOP for Javascript :) 

I'll merge this PR, but I just share some of my thoughts!